### PR TITLE
[Resolve #1] Display local albums

### LINKED ISSUE #1: Display local albums
when user click the "choose albums" button on home view,

- pop up a view contains all the local albums
- thumbnail on left
- album name on right

### PR DESCRIPTION
**Pull request by Snow-Zyetric**

### CODE REVIEW

**Review by zteric42 — CHANGES_REQUESTED** [review]
remove file header comments

File: AlbumApp/ViewController.swift [hunk 1]
```diff
@@ -1,15 +1,58 @@
...
-//  Created by 张雪 on 19/09/17.
-//  Copyright © 2017 张雪. All rights reserved.
+//  Created by 张雪 on 2017/8/24.
+//  Copyright © 2017年 张雪. All rights reserved.
 //
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UIViewController, UINavigationControllerDelegate,UIImagePickerControllerDelegate {
```
Comment (zteric42) [inline AlbumApp/ViewController.swift:11]: use class extension

**Review by zteric42 — CHANGES_REQUESTED** [review]
remove file header comments

File: AlbumApp/ViewController.swift [hunk 2]
```diff
@@ -1,15 +1,58 @@
...
 
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var photoImage: UIImageView!
+    
+    
+    @IBAction func importImage(_ sender: Any) {
+        let image = UIImagePickerController()
```
Comment (zteric42) [inline AlbumApp/ViewController.swift:18]: rename better name

File: AlbumApp/ViewController.swift [hunk 3]
```diff
@@ -1,15 +1,58 @@
...
+        image.delegate = self
+        image.sourceType = UIImagePickerControllerSourceType.photoLibrary
+        
+        image.allowsEditing = false
+        self.present(image, animated: true)
+    }
+    
+    @IBAction func takePhoto(_ sender: Any) {
+        let tkP = UIImagePickerController()
```
Comment (zteric42) [inline AlbumApp/ViewController.swift]: rename better name

**Review by zteric42 — APPROVED** [review]

File: AlbumApp/ViewController.swift [hunk 4]
```diff
@@ -1,15 +1,58 @@
...
+    
+    @IBAction func takePhoto(_ sender: Any) {
+        let imagePicker = UIImagePickerController()
+        imagePicker.delegate = self
+        imagePicker.sourceType = UIImagePickerControllerSourceType.camera
+        imagePicker.allowsEditing = false
+        self.present(imagePicker, animated: true)
+        
+        
+        //fake fixed
```
Comment (zteric42) [inline AlbumApp/ViewController.swift:34]: remove debug info

**Review by zteric42 — CHANGES_REQUESTED** [review]

File: AlbumApp/ViewController.swift (hunk 4 above)
Comment (Snow-Zyetric) [reply AlbumApp/ViewController.swift:34]: will remove after whole feature

**Pull request merged**